### PR TITLE
Claim finished tasks when "started" event delays

### PR DESCRIPTION
**Pull request by rail**
This should address the "fast task" issue, when we get the "started" event after "log_upload" for some reason.

### CODE REVIEW

**Comment by rail** [comment]
@catlee what do you think?

File: bbb/services.py [hunk 1]
```diff
@@ -160,7 +160,16 @@ def _handleFinishedRequest(self, brid, properties, results):
...
+        log.info("Handling finished task %s run %s brid %s", taskid, runid,
+                 brid)
+        # Try to claim the task in case if the "started" event comes after
+        try:
+            self.tc_queue.claimTask(taskid, runid, {
+                "workerGroup": self.tc_worker_group,
+                "workerId": self.tc_worker_id,
+            })
+        except TaskclusterRestFailure:
+            log.info("Cannot claim task %s run %s, assuming it is claimed already", taskid, runid)
```
Comment (catlee) [inline bbb/services.py]: My only concern here is that the logging is going to be pretty noisy. In the normal case we're going to have a lot of "Cannot claim task ..." messages, which kind of sounds like an error. Maybe turn this into a log.debug() call, and have a log.info() call if we successfully claimed the task above?

**Comment by rail** [comment]
should be better now

**Comment by catlee** [comment]
:sheep: it

**Pull request merged**